### ISSUE TITLE
In some cases, inline attachments are not shown in the body - sent with Apple Mail

### ISSUE THREAD

**Issue by Artem-Schander** [barbushin/php-imap#174]
Thanks for the package. Nice work.
Just one little thing; 
while i'm sending a normal HTML email (without additional attachments), all images within my signature are taking place where they belong. Also all inline image attachments are working well. But as soon as i attach a PDF or ZIP or something, all attached inline images getting removed from the HTML.

Maybe this [StackOverflow question](http://stackoverflow.com/questions/11285430/imap-get-attached-file) helps. 
However, I'll fork your package in a few hours and try to fix it. But maybe you can help me anyway?

`without attached PDF`
![bildschirmfoto 2016-11-30 um 10 48 41](https://cloud.githubusercontent.com/assets/1243826/20747585/e15b64d4-b6ea-11e6-8fe3-fc0bfaa0b13a.png)

`with attached PDF`
![bildschirmfoto 2016-11-30 um 10 49 11](https://cloud.githubusercontent.com/assets/1243826/20747592/e5ffabda-b6ea-11e6-891e-1393856db109.png)

**Comment by barbushin** [comment]
Could you please send this email to barbushin3@gmail.com, so I can reproduce this problem and fix? Thank you.

**Comment by Artem-Schander** [comment]
done

**Comment by Sebi94nbg** [comment]
Hi,
First of all: Thank you for this great package! I tried to build my own functions to save emails in the database and it's just terrible difficult with all of these different encodings, structures and so on. Good job!

I also store HTML and PlainText emails in a database in order to display them later in a web based system.

How can I store emails with embedded inline images in the database and show them on the web based system as it usually was?

In my case, I've saved the `$email->textHtml` in the database and it displays the email in the web based system as expected, but the image is broken - it only shows the broken image thumbnail (but in the original size of the image).

Is there a way to download the inline image, save it as file on the storage and replace the embedded inline image with a linked text to the downloaded image? Or can I somehow save the inline image as base64 data in order to reuse it with the `<img>` tag?

Thanks in advance!
